dpe/src/x509.rs

```diff
@@ -489,7 +489,7 @@ impl CertWriter<'_> {
         tagged: bool,
         is_x509: bool,
     ) -> Result<usize, DpeErrorCode> {
-        if !measurements.is_ca || !is_x509 {
```

Comment:
Based on my read of RFC5280, I think we can keep this check for SKI. Just not for AKI. wdyt?

Reply:
SGTM!